from django.conf import settings

USE_SUBDOMAIN_EXCEPTION = getattr(settings, 'USE_SUBDOMAIN_EXCEPTION', False)